desktop/src/main/java/haveno/desktop/main/account/content/seedwords/SeedWordsView.java

```diff
@@ -106,6 +106,8 @@ protected void initialize() {
         displaySeedWordsTextArea.setEditable(false);
 
         datePicker = addTopLabelDatePicker(root, ++gridRow, Res.get("seed.date"), 10).second;
+        //note: on MacOS this datePicker does not actually list a date...is it
```

Comment:
Please remove this comment since it's a separate bug which will be fixed separately.

Reply:
The comment is now removed

Reply:
The code is still using JFXDatePicker in add2TopLabelDatePicker. Thinking that should be switched over too?